src/recipes.js

```diff
@@ -1,180 +1,235 @@
                         },
                         {
                             "ingredient_name" : "b", 
-                            "ingredient_quantity" : "amount"
+                            "ingredient_quantity" : "amount", 
+                            "ingredient_img" : "https://i.cbc.ca/1.1894364.1380814651!/fileImage/httpImage/image.jpg_gen/derivatives/16x9_940/sugar.jpg"
                         },
                         {
                             "ingredient_name" : "c", 
-                            "ingredient_quantity" : "amount"
+                            "ingredient_quantity" : "amount", 
+                            "ingredient_img" : "https://i.cbc.ca/1.1894364.1380814651!/fileImage/httpImage/image.jpg_gen/derivatives/16x9_940/sugar.jpg"
                         }
                     ],
                     "instructions" : [
                         "do x", 
                         "then do y"
                     ]
-                },{
+                },
+                "Rice-Side-Dishes-2-recipe-2": {
                     "name" : "Rice Side Dishes 2 recipe 2",
                     "img_url" : "https://images-prod.healthline.com/hlcmsresource/images/AN_images/healthy-eating-ingredients-1296x728-header.jpg",
+                    "key_ingredients" : ["ing 1", "ing 2"],
                     "ingredients" : [
                         {
                             "ingredient_name" : "a", 
-                            "ingredient_quantity" : "amount"
+                            "ingredient_quantity" : "amount", 
+                            "ingredient_img" : "https://i.cbc.ca/1.1894364.1380814651!/fileImage/httpImage/image.jpg_gen/derivatives/16x9_940/sugar.jpg"
                         },
                         {
                             "ingredient_name" : "b", 
-                            "ingredient_quantity" : "amount"
+                            "ingredient_quantity" : "amount", 
+                            "ingredient_img" : "https://i.cbc.ca/1.1894364.1380814651!/fileImage/httpImage/image.jpg_gen/derivatives/16x9_940/sugar.jpg"
                         },
                         {
                             "ingredient_name" : "c", 
-                            "ingredient_quantity" : "amount"
+                            "ingredient_quantity" : "amount", 
+                            "ingredient_img" : "https://i.cbc.ca/1.1894364.1380814651!/fileImage/httpImage/image.jpg_gen/derivatives/16x9_940/sugar.jpg"
                         }
                     ],
                     "instructions" : [
                         "do x", 
                         "then do y"
                     ]
-                }]
+                }}
             
 };
 
+export function getRecipe(recipeId) {
+    for (const [key] of Object.entries(allRecipes)) {
+        if(allRecipes[key].hasOwnProperty(recipeId))
+            return allRecipes[key][recipeId];
+    }
+    return "error";
+}
+
```

Comment:
I might not be following this correctly, but since your `allRecipes` object is keyed by the recipe ID, would the following achieve the same thing:

```js
export function getRecipe(recipeId) {
  return allRecipes[recipeId] || 'error';
}